test/cri-containerd/main.go

```diff
@@ -149,6 +149,13 @@ func requireBinary(t *testing.T, binary string) string {
 }
 
 func getWindowsNanoserverImage(build uint16) string {
+	// Due to some efforts in improving down-level compatibility for Windows containers (see
+	// https://techcommunity.microsoft.com/t5/containers/windows-server-2022-and-beyond-for-containers/ba-p/2712487)
+	// the ltsc2022 image should continue to work on builds ws2022 and onwards. The panic for "unsupported build"
+	// should only be triggered if the user is on a build older than RS5 or 21h1 now.
```

Comment:
Nit: "on 21h1 or a build older than RS5 now". The way it reads now, "21h1" easy reads as part of "a build older than".

Reply:
That makes more sense to me, changing!

Reply:
Done, thanks again